packages/eslint-config-fbjs/package.json

```diff
@@ -26,6 +26,6 @@
     "eslint-plugin-flowtype": "^2.35.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.1.0",
-    "eslint-plugin-relay": "0.0.8"
+    "eslint-plugin-relay": ">=0.0.8 <=0.0.17"
```

Comment:
Let's just make this `^0.0.8` and that way it'll pick up future `0.0.z` versions. The risk that that they make a breaking change in there but we can deal with that if it happens.

Reply:
Sounds right.

Reply:
This is wrong, you have to do `~0.0.8`.

![image](https://user-images.githubusercontent.com/1404810/31573125-64be9ffc-b0b5-11e7-8c47-cac49f468c47.png)
![image](https://user-images.githubusercontent.com/1404810/31573127-70aa2160-b0b5-11e7-9c6f-2766fb098e7d.png)

https://semver.npmjs.com/

Reply:
#271